From 592c6bc3e5e93f36c2fdc0a491a9fb43a41cf688 Mon Sep 17 00:00:00 2001
From: Fabian H <fabolhak@users.noreply.github.com>
Date: Tue, 17 Aug 2021 00:52:39 +0200
Subject: [PATCH] add missing - in certopt (#210)

otherwise openssl will give an error and not accept is as argument
---
 rsync-ssl | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/rsync-ssl b/rsync-ssl
index 46701af1..8588eeb0 100755
--- a/rsync-ssl
+++ b/rsync-ssl
@@ -73,7 +73,7 @@ function rsync_ssl_helper {
 	certopt=""
 	gnutls_cert_opt=""
     else
-	certopt="cert$optsep$RSYNC_SSL_CERT"
+	certopt="-cert$optsep$RSYNC_SSL_CERT"
 	gnutls_cert_opt="--x509keyfile=$RSYNC_SSL_CERT"
     fi
 
